notebooks/community/CODEOWNERS

```diff
@@ -17,6 +17,7 @@
 /explainable_ai/SDK_Custom_Container_XAI.ipynb @brianchunkang
 /matching_engine/sdk_matching_engine_for_indexing.ipynb @ivanmkc
 /matching_engine/matching_engine_for_indexing.ipynb @yinghsienwu
+/matching_engine/stream_update_for_matching_engine.ipynb @peterping
```

Comment:
This has to be your Github username: `peterping666`

Reply:
ok